src/components/header/header.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Icon } from 'react-fa';
+import './header.scss';
+import Logo from '../../components/logo/logo';
+import SlackLogo from '../../images/Slack-Monochrome-White.svg';
+
+export default class Hello extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className="row-fluid">
+        <div id="header" className="col-xs-12">
+          <div className="col-xs-2 logo">
+            <Logo />
+          </div>
+
+          <div className="col-xs-7">&nbsp;</div>
+
+          <div className="social col-xs-3">
```

Comment:
fwiw, the social icons break out of a container this small. You might want to give it a bigger container for smaller viewports, and adjust on larger viewports: 

![image](https://user-images.githubusercontent.com/2982692/31676015-86e97b4a-b323-11e7-9fa0-18b6142bff27.png)